Gruntfile.js

```diff
@@ -362,7 +370,7 @@ module.exports = (grunt) => {
 
   grunt.registerTask('default', ['dev'])
   // TODO: add 'deploy' as per https://github.com/okTurtles/group-income/issues/10
-  grunt.registerTask('dev', ['checkDependencies', 'backend:relaunch', 'build:watch', 'keepalive'])
+  grunt.registerTask('dev', ['checkDependencies', 'build:watch', 'backend:relaunch', 'keepalive'])
```

Comment:
Why were these re-arranged?

Reply:
Previously, the backend would have finished launchng quite long before the new frontend assets were ready, so the client would already reconnect but get the old bundle when reloading the page.

Reply:
Hmm. OK, as long as it doesn't break anything. Changes like these make me nervous as I feel there was a rhyme & reason to the order...

Reply:
I knew my spidy-sense was tingling!

<img width="1294" alt="Screen Shot 2022-04-03 at 4 42 09 PM" src="https://user-images.githubusercontent.com/138706/161454063-c285122b-c6f2-43e4-a1ab-10eb372ffe08.png">